obidog/wrappers/doxygen_wrapper.py

```
@@ -4,9 +4,33 @@
 import tempfile
 
 from obidog.config import SOURCE_DIRECTORIES
+from obidog.logger import log
 
 
+DOXYGEN_PATH = os.environ.get("DOXYGEN_PATH", "doxygen")
+
+def _check_doxygen():
+    try:
+        with subprocess.Popen(
+            [DOXYGEN_PATH, "--version"], stdout=subprocess.PIPE
+        ) as clang_format_exec:
+            version = clang_format_exec.stdout.read().decode("utf-8").strip()
+            try:
+                version = version.split()[0].split(".")
+                if int(version[0]) >= 1 and int(version[1]) >= 8:
+                    return True
+                else:
+                    return False
+            except:
+                return False
+    except FileNotFoundError as e:
+        return False
+
 def build_doxygen_documentation(source_path):
+    if DOXYGEN_PATH is None:
+        log.warn("doxygen not found, could not create doxygen files")
```

Doxygen is mandatory, you can raise an Exception if it wasn't found